CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Changelog
 
+## v0.31.0
+
+*March 13th, 2019*
+
+Special thanks to external contributors on this release:
+@danil-lashin, @guagualvcha, @jleni, @siburu, @silasdavis, @srmo, @Stumble, @svenstaro 
+
+### BREAKING CHANGES:
+
+* CLI/RPC/Config
+- [rpc/client] Update Subscribe interface to reflect new pubsub/eventBus API [ADR-33](https://github.com/tendermint/tendermint/blob/develop/docs/architecture/adr-033-pubsub.md)
```

Comment:
- [config] Add `rpc.max_subscription_clients` config parameter to control how many unique clientIDs that can /subscribe at the same time
- [config] Add `rpc.max_subscriptions_per_client` config parameter to control how many unique queries a given client can /subscribe to
- [config] Add `rpc.max_subscription_clients` config parameter allowing you to change time  to wait for a tx to be committed during /broadcast_tx_commit